packages/extras/src/lib/components/controls/OrbitControls/OrbitControls.svelte

```diff
@@ -31,17 +31,25 @@
   })
 
   const { orbitControls } = useControlsContext()
-  orbitControls.set(controls)
-  onDestroy(() => orbitControls.set(undefined))
+
+  $effect(() => {
+    const handleChange = (event: Event) => {
+      invalidate()
+      props.onchange?.(event)
+    }
+
+    orbitControls.set(controls)
+    controls.addEventListener('change', handleChange)
+    return () => {
+      orbitControls.set(undefined)
+      controls.removeEventListener('change', handleChange)
+    }
+  })
```

Comment:
This fixes a bug in which OrbitControls was overwriting `onchange` listeners in some circumstances (like the gizmo example)

Reply:
i think i have the same sort of fix in my local experimenting branch for #1237